### PR TITLE
Reduce cyclometric complexity of createReport function

### PR DESCRIPTION
**Pull request by briankfitzwater**
- creates a struct for environment variables to pass to Inv.New(env EnvVars)
- adds a function to validate the environment variables (validateEnvVars)
- reduces cyclometric complexity of createReport function

### CODE REVIEW

**Comment by briankfitzwater** [comment]
[[DEV] Create a modular customer provisioning process with validation mechanism (Break-Down)](https://trello.com/c/eTIoCByg/983-dev-create-a-modular-customer-provisioning-process-with-validation-mechanism-break-down)

**Comment by briankfitzwater** [comment]
[[DEV] Reduce cyclomatic complexity of grace-inventory-lambda.createReport function](https://trello.com/c/pe6dw9Vm/1041-dev-reduce-cyclomatic-complexity-of-grace-inventory-lambdacreatereport-function)

File: handler/helpers/helpers.go [hunk 1]
```diff
@@ -44,6 +68,61 @@ func Accounts(cfg client.ConfigProvider) ([]*organizations.Account, error) {
...
+		&s3.GetObjectInput{
+			Bucket: aws.String(bucket),
+			Key:    aws.String(item),
+		})
+
+	if err != nil {
+		return nil, err
+	}
+
+	var dat map[string]interface{}
```
Comment (bryanlalexander) [inline handler/helpers/helpers.go]: Is this a non-standard data structure? This should be unmarshalling into a structure. (guess this didn't submit last time)

File: handler/main.go [hunk 2]
```diff
@@ -24,32 +25,45 @@ func getEnv(name string) (string, error) {
 	return val, nil
 }
 
-func createReport() (string, error) {
-	bucket, err := getEnv(envBucket)
+func validateEnvVars() (inv.EnvVars, error) {
```
[the remark this answers is not in the document]
Comment (briankfitzwater) [reply handler/main.go]: That looks much better.

**Pull request closed without merging**